pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -433,6 +468,7 @@ func TestTaskSpecValidate(t *testing.T) {
 			}
 			ctx := config.EnableAlphaAPIFields(context.Background())
 			ts.SetDefaults(ctx)
+			ctx = context.WithValue(ctx, v1beta1.ValidateParameterVariablesAndWorkspaces, false)
```

Comment:
curious why any of the tests other than the ones with parameter propagation have validation = false?